tls/s2n_connection.h

```diff
@@ -86,6 +86,9 @@ struct s2n_connection {
     uint8_t session_id[S2N_TLS_SESSION_ID_MAX_LEN];
     uint8_t session_id_len;
 
+    /* Session resumption indicator on client side */
+    unsigned int client_session_resumed:1;
```

Comment:
lets keep our bit fields close together - please move this up next to ```corked_io:1```

others:

```
unsigned int close_notify_queued:1;
unsigned int secure_renegotiation:1;
```

and you can drop the ```int``` too.